tools/gulptasks/scripts-ts.js

```diff
@@ -252,6 +252,19 @@ async function scriptsTS(argv) {
                 );
             }
 
+            const dashboardsBuildPropertiesJSON =
+                fsLib.getFile(fsLib.path(['tools', 'gulptasks', 'dashboards', 'build-properties.json']), true);
+            const dashboardsEsmFiles =
+                fsLib.getFilePaths(fsLib.path(['code', 'dashboards', 'es-modules']), true);
+
+            for (const file of dashboardsEsmFiles) {
+                await replaceProductPlaceholders(
+                    file,
+                    'Dashboards',
+                    argv.release || dashboardsBuildPropertiesJSON.version
```

Comment:
For Grid and Dash, we rely solely on the `buildProperties`, so I would remove the `argv.release`.